xml/admin_gui_dashboard.xml

```diff
@@ -953,9 +954,108 @@
     </para>
     <para>
      Click the down arrow in the top left of the table heading next to the
-     <guimenu>Edit</guimenu> button and select the task you want to perform.
+     <guimenu>Create</guimenu> button and select the task you want to perform.
     </para>
    </tip>
+   <sect3 xml:id="dashboard-cluster-osds-add">
+    <title>Adding OSDs</title>
+    <para>
+     To add new OSDs, follow these steps:
+    </para>
+    <procedure>
+     <step>
+      <para>
+       Verify that there are storage devices <emphasis>available</emphasis> on
+       cluster nodes (to see when a device is <emphasis>available</emphasis>,
+       see <xref linkend="deploy-cephadm-day2-service-osd"/>). Then click the
+       down arrow in the top left of the table heading and select
+       <guimenu>Create</guimenu>. This opens the <guimenu>Create OSDs</guimenu>
+       window.
+      </para>
+      <figure>
+       <title>Create OSDs</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="dash_osd_add.png" width="75%"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="dash_osd_add.png" width="75%"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+     </step>
+     <step>
+      <para>
+       Click <guimenu>Add</guimenu> to add primary storage devices for OSDs,
+       and specify filtering criteria in the top right of the <guimenu>Primary
+       devices</guimenu> table, for example <guimenu>Type</guimenu>
+       <guimenu>hdd</guimenu>. Confirm with <guimenu>Add</guimenu>.
```

Comment:
```suggestion
       To add primary storage devices for OSDs, click <guimenu>Add</guimenu>. 
       You can also specify filtering criteria in the top right of the <guimenu>Primary
       devices</guimenu> table&mdash;for example <guimenu>Type</guimenu>
       <guimenu>hdd</guimenu>. Confirm with <guimenu>Add</guimenu>.
```

Reply:
Again, breaking a long sentence into 2, and putting the action first and how to do it 2nd.